presentation/src/main/java/com/example/pinterbest/presentation/viewmodels/CreatorsViewModel.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.pinterbest.presentation.viewmodels
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.pinterbest.domain.common.Result
+import com.example.pinterbest.domain.entities.Profile
+import com.example.pinterbest.domain.usecases.GetPinsCreatorsUseCase
+import javax.inject.Inject
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+
+class CreatorsViewModel @Inject constructor(
+    private val getPinsCreatorsUseCase: GetPinsCreatorsUseCase
+) : ViewModel() {
+    private val _state = MutableLiveData(false)
+    val state: LiveData<Boolean> = _state
```

Comment:
`state` of what? Each name should be self documented. Ideally, anybody who sees your code should clearly understand what exactly does variable/class/method without further code exploration. Let's add a little bit of context into the name